Contents/Sketch/typography.js

```diff
@@ -7,7 +7,10 @@ var applyTypography = function (newStyles, sharedStyles) {
     'justified': 3
   };
 
-  removeAllStyles();
```

Comment:
I can't decide what to do on this one. Neither of the solutions seem ideal. I might just leave it the way you had it.

I'm almost finished with a new feature that will:
1. look for any existing text styles that match style names in the spreadsheet
2. then update them with any changes in the spreadsheet
3. then append any additional styles